rastervision/runner/command_definition.py

```diff
@@ -36,12 +40,14 @@ def from_experiments(cls, experiments: List[rv.ExperimentConfig]):
             for command_type in rv.ALL_COMMANDS:
                 log.debug(
                     'Updating config for command {}...'.format(command_type))
-                io_def = e.update_for_command(command_type, e)
+                e.update_for_command(command_type, e)
                 log.debug('Creating experiment configuration...'.format(
                     command_type))
-                command_config = e.make_command_config(command_type)
-                command_def = cls(e.id, command_config, io_def)
-                command_definitions.append(command_def)
+                base_command_config = e.make_command_config(command_type)
+                for command_config in base_command_config.split(splits):
+                    io_def = command_config.report_io()
+                    command_def = cls(e.id, command_config, io_def)
+                    command_definitions.append(command_def)
```

Comment:
This is the  crux of the changes - by separating the io_def updates and the update_for_command, we can generate commands from the fully resolved and unsplit experiment, and then split the commands, then ask the commands to provide the io_def updates.